static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -0,0 +1,21 @@
+# How to update your pipeline by editing dvc files
+
+> 1)using `dvc move` to edit file paths of outputs or dependencies.
+> 2)editting by hand the outputs paths.
```

Comment:
space missing. Use capital letters to start list items if they contain full sentences.

Reply:
also, editting -> editing